taxi/views.py

```diff
@@ -85,6 +117,19 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs) -> dict:
+        context = super(DriverListView, self).get_context_data(**kwargs)
+        search = self.request.GET.get("search", "")
+        context["search_form"] = SearchForm(initial={"search": search})
+        return context
+
+    def get_queryset(self) -> QuerySet:
+        queryset = Driver.objects.all()
+        filter_ = self.request.GET.get("search")
+        if filter_:
+            return queryset.filter(username__icontains=filter_)
+        return queryset
```

Comment:
1. Do you need to define class in super when only one parent class has this method?
2. Call super method to get an initial queryset
fix everywhere